tools/appveyor.psm1

```diff
@@ -499,6 +509,15 @@ function Invoke-AppveyorFinish
         }
         Write-Log "MSI smoke test was successful"
 
+        # Verify path was updated by MSI
+        $psPath = ([System.Environment]::GetEnvironmentVariable('PATH','MACHINE')) -split ';' |
```

Comment:
Wouldn't the env var only be updated for a new process?

Reply:
that is why I'm reading it from the machine

Reply:
got it.  thanks